api/app/signals/apps/email_integrations/templates/email/signal_status_changed.txt

```diff
@@ -0,0 +1,26 @@
+Geachte melder,
```

Comment:
mail template name suggestion: something in the vain of `signal_status_changed_afgehandeld.txt`

Reply:
done